modules/appeals_api/spec/support/doc_helpers.rb

```diff
@@ -188,10 +188,8 @@ def self.api_tags
   def self.api_base_path_template
     if DocHelpers.decision_reviews?
       '/services/appeals/{version}/decision_reviews'
-    elsif DocHelpers.api_name == 'appeals_status'
-      '/services/appeals/{version}'
```

Comment:
Note that the older appeals status API v0 docs are not rendered via rswag, so they're not a case to worry about here